samples/c2pa/C2paPlayer/C2paMenu/C2paMenuFunctions.js

```diff
@@ -98,19 +121,55 @@ export let updateC2PAMenu = function (c2paStatus, c2paMenu , isMonolithic , vide
         );
 
         if (c2paItemValue != null) {
+            //formatting for social media links
+            if(c2paItemKey=== 'SOCIAL' ){
+                var socialArray = c2paItemValue.map(function (account) {
+                    var formattedWebsite = providerInfoFromSocialId(account).name
+                    return `<span><a class="url" href="${account}" onclick="window.open('${account}')">${formattedWebsite}</a></span>`
+                });
+                c2paItem.el().innerHTML =
+                '<span class="itemName nextLine">' +
+                c2paItemName +
+                '</span>' +
+                c2paMenuInstance.c2paMenuDelimiter() + socialArray.join('\n')
+            }
+            //styling alerts
+            else if(c2paItemKey=== 'ALERT' ){
+                c2paItem.el().innerHTML = '<div class="alert-div alerting">' +'<img class="alert-icon"></img>'+ '<div>' +
```

Comment:
Should we prefix these classes so they don't clash with other things on the page? Also, is concatenation needed here, or just for readability? It might be nicer to just do something like:
```js
c2paItem.el().innerHTML = `
    <div class="alert-div alerting">
        <img class="alert-icon" />
         ...
    </div>
`;
```